packages/next-auth/package.json

```diff
@@ -120,7 +120,7 @@
     "jest-environment-jsdom": "^28.1.1",
     "jest-watch-typeahead": "^1.1.0",
     "msw": "^0.42.3",
-    "next": "13.0.6",
+    "next": "13.2.1",
```

Comment:
13.3.0 should work, we should also update our peer dependency requirement at line 81 to at least 13.3.0